app/common/verifier/verifier.js

```diff
@@ -273,8 +273,14 @@ const isCheckDisabled = function (env, name) {
 };
 
 const checkStructure = function (a, b, url) {
+    delete a.$promise;
+    delete a.$resolved;
+    delete b.$promise;
+    delete a.$resolved;
```

Comment:
@rowasc Ugh this looks not so nice, do you have any other idea?

Reply:
underscore's _filter maybe ?